pkg/model/bootstrapscript.go

```diff
@@ -221,7 +285,12 @@ func (b *BootstrapScript) ResourceNodeUp(ig *kops.InstanceGroup, cluster *kops.C
 		return nil, err
 	}
 
-	templateResource, err := NewTemplateResource("nodeup", awsNodeUpTemplate, functions, nil)
+	resoureFunction := functions
+	if ig.IsBastion() {
+		resoureFunction = bastionFunctions
```

Comment:
Ah - it gets even trickier now, because nodeup previously didn't run on the bastions.  But now it looks like you are running it - that's a pretty big change, which I'm not fundamentally opposed to, but we need to be very careful about.

Reply:
My use case need the bastions as well, if you don't want it , we can roll this back